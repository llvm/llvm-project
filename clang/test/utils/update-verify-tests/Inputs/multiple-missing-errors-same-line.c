void foo() {
    a = 2; b = 2; c = 2;
}

void bar() {
    x = 2; y = 2; z = 2;
    // expected-error@-1{{use of undeclared identifier 'x'}}
}
