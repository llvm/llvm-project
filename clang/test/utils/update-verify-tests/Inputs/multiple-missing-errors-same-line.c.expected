void foo() {
    // expected-error@+3{{use of undeclared identifier 'c'}}
    // expected-error@+2{{use of undeclared identifier 'b'}}
    // expected-error@+1{{use of undeclared identifier 'a'}}
    a = 2; b = 2; c = 2;
}

void bar() {
    x = 2; y = 2; z = 2;
    // expected-error@-1{{use of undeclared identifier 'x'}}
    // expected-error@-2{{use of undeclared identifier 'y'}}
    // expected-error@-3{{use of undeclared identifier 'z'}}
}
